Makefile

```diff
@@ -0,0 +1,52 @@
+REPO_URL="xpkg.upbound.io/upbound/function-shell"
```

Comment:
this repo URL should now point to crossplane-contrib

Reply:
Updated. Will be in next PR.